src/runtime/plugin.ts

```diff
@@ -17,7 +18,8 @@ export default defineNuxtPlugin(async () => {
     console.error(e);
   }
 
-  if (options.sessions.refreshEvery) {
+  // eslint-disable-next-line
+  if (!!options.sessions.refreshEvery) {
```

Comment:
Hope that `!!` fil fix the issue

![image](https://github.com/LorexIQ/nuxt-local-auth/assets/29500153/42e02d5c-2ecb-4246-8e5a-7f531749501f)